cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,39 @@
+import { flags } from '@oclif/command'
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+
+export default class MembershipValidatorAccountCommand extends WorkingGroupsCommandBase {
```

Comment:
Maybe rename this class to something like `VerifyValidatorAccountCommand` to better reflect the usecase?